pyproject.toml

```diff
@@ -26,6 +27,14 @@ ignore = [
     "E741",
 ]
 target-version = "py37"
+exclude = [
+  "src/taskgraph/run-task/robustcheckout.py",
+  "taskcluster/scripts/external_tools/tooltool.py",
```

Comment:
Maybe we should just exclude all of `external_tools` as presumably we wouldn't want to lint anything in there.